### PR TITLE
Corrected variable checks for talk_impressions_of_the_other_girls

### PR DESCRIPTION
**Pull request by Blizzardsev**
- Fixes issue for talk_impressions_of_the_other_girls, where Natsuki will not handle repeat conversations correctly in conditioning.
- Clarify tons of readme content

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,31 +163,33 @@ Please note that until automatic updates are implemented, any updates released m
...
-7. **Copy** the `game` and `lib` folders from your **fresh DDLC folder** into your JN installation.
-8. From the release files, copy the `game` folder contents (all of it) into the `game` folder of your JN installation. If asked to replace files, select _Yes_.
-9. From the release files, copy the `lib` folder contents (all of it) into the `lib` folder of your JN installation. If asked to replace files, select _Yes_.
-10. From the release files, copy the `update` folder into your JN installation.
-11. Finally, launch `DDLC.exe`.
+*Use these instructions if you have a BETA version of JN you are updating to a RELEASE (1.0.0 onwards).*
+
+1. Stop JN, if you are currently playing the mod.
+2. **Back up your persistent**: instructions on how to do so may be found [here](https://github.com/Just-Natsuki-Team/NatsukiModDev/wiki/04:-FAQ#can-i-back-up-my-save-data--how-do-i-find-my-persistent).
+3. Download a fresh copy of DDLC from https://ddlc.moe. **Do not use a Steam installation**.
```
Comment (TRIDENT1313) [inline README.md:170]: Do we want to link to the site, as opposed to just writing it?
```suggestion
3. Download a fresh copy of DDLC from [https://ddlc.moe](https://ddlc.moe). **Do not use a Steam installation**.
```
Alternatively...
```suggestion
3. Download a fresh copy of DDLC from the [official site](https://ddlc.moe). **Do not use a Steam installation**.
```

File: README.md [hunk 2]
```diff
@@ -163,31 +163,33 @@ Please note that until automatic updates are implemented, any updates released m
...
 
-1. **Back up your persistent**: instructions on how to do so may be found [here](https://github.com/)
-2. Download the release files: go to https://github.com/Just-Natsuki-Team/NatsukiModDev, then select the most recent release under _Releases_. Select the ZIP file starting with `jn` (I.E `jn-1.0.0.zip`).
-3. Extract/unzip the new release files.
-4. From the release files, copy the `game` folder contents (all of it) into the `game` folder of your JN installation. If asked to replace files, select _Yes_.
-5. From the release files, copy the `lib` folder contents (all of it) into the `lib` folder of your JN installation. If asked to replace files, select _Yes_.
-6. From the release files, copy the `update` folder contents (all of it) into the `update` folder of your JN installation. If asked to replace files, select _Yes_.
-7. Finally, launch `DDLC.exe`.
+1. Stop JN, if you are currently playing the mod.
+2. **Back up your persistent**: instructions on how to do so may be found [here](https://github.com/)
```
Comment (TRIDENT1313) [inline README.md:186]: Why are we linking straight to the git homepage?

File: README.md [hunk 3]
```diff
@@ -163,31 +163,33 @@ Please note that until automatic updates are implemented, any updates released m
...
+3. Download the release files: go to https://github.com/Just-Natsuki-Team/NatsukiModDev, then select the most recent release under _Releases_. Select the ZIP file starting with `jn` (I.E `jn-1.0.0.zip`).
```
Comment (TRIDENT1313) [inline README.md:187]: Maybe link to it?
Saves copy-pasting...
```suggestion
3. Download the release files: go to [https://github.com/Just-Natsuki-Team/NatsukiModDev](https://github.com/Just-Natsuki-Team/NatsukiModDev), then select the most recent release under _Releases_. Select the ZIP file starting with `jn` (I.E `jn-1.0.0.zip`).
```

File: README.md [hunk 4]
```diff
@@ -163,31 +163,33 @@ Please note that until automatic updates are implemented, any updates released m
...
-9. From the release files, copy the `lib` folder contents (all of it) into the `lib` folder of your JN installation. If asked to replace files, select _Yes_.
-10. From the release files, copy the `update` folder into your JN installation.
-11. Finally, launch `DDLC.exe`.
+*Use these instructions if you have a BETA version of JN you are updating to a RELEASE (1.0.0 onwards).*
+
+1. Stop JN, if you are currently playing the mod.
+2. **Back up your persistent**: instructions on how to do so may be found [here](https://github.com/Just-Natsuki-Team/NatsukiModDev/wiki/04:-FAQ#can-i-back-up-my-save-data--how-do-i-find-my-persistent).
+3. Download a fresh copy of DDLC from https://ddlc.moe. **Do not use a Steam installation**.
+4. Extract/unzip DDLC.
+5. Download the release: go to https://github.com/Just-Natsuki-Team/NatsukiModDev, then select the most recent release under _Releases_. Select the ZIP file starting with `jn` (I.E `jn-1.0.0.zip`).
```
Comment (TRIDENT1313) [inline README.md:172]:
```suggestion
5. Download the release: go to [https://github.com/Just-Natsuki-Team/NatsukiModDev](https://github.com/Just-Natsuki-Team/NatsukiModDev), then select the most recent release under _Releases_. Select the ZIP file starting with `jn` (I.E `jn-1.0.0.zip`).
```

**Review by TRIDENT1313 — APPROVED** [review]
Just a couple of suggestions.
Aside from that, nothing major, happy to approve as is.

File: README.md (hunk 1 above)
Comment (Blizzardsev) [reply README.md:170]: This is a good catch; adding

File: README.md (hunk 2 above)
Comment (Blizzardsev) [reply README.md:186]: We should not be - should be pointing to the FAQ segment on persistent backups. Will get that sorted

File: README.md (hunk 3 above)
Comment (Blizzardsev) [reply README.md:187]: We can do one better and simply link to the releases: less verbose and quicker to do

File: README.md (hunk 4 above)
Comment (Blizzardsev) [reply README.md:172]: We can do one better and simply link to the releases: less verbose and quicker to do

**Review by TRIDENT1313 — APPROVED** [review]
Looks good to me!

**Pull request merged**